html/disabled/demos/base/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Примеры использования — Атрибут `disabled` — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    *, *::before, *::after {
+      margin: 0;
+      padding: 0;
+      box-sizing: border-box;
+    }
+
+    html, body {
+      min-height: 100%;
+    }
+
+    body {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      padding: 50px 15px;
+      font-family: "Roboto", sans-serif;
+      font-size: 24px;
+      color: #ffffff;
+      accent-color: darkmagenta;
+      background-color: #18191c;
+    }
+
+    div {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 25px;
+      width: 100%;
+      max-width: 650px;
+      margin: 0 auto;
+      padding: 10px;
+      border: 1px solid #683612;
+    }
+
+    div:not(:last-child) {
+      margin-bottom: 30px;
+    }
+
+    button,
+    input,
+    textarea,
+    select {
+      font-family: inherit;
+      font-size: inherit;
+    }
+
+    button, fieldset, select, textarea {
+      padding: 10px 15px;
+    }
```

Comment:
```suggestion
    button,
    input,
    textarea,
    select {
      font: inherit;
    }

    button,
    fieldset,
    select,
    textarea {
      padding: 10px 15px;
    }
```

Выровнял стиль кода, упростил наследование шрифта